app/src/components/SingleDatePicker/SingleDatePicker.js

```diff
@@ -0,0 +1,109 @@
+import React, { useCallback, useMemo, useRef, useState } from 'react'
+import PropTypes from 'prop-types'
+import { Popover, GU, RADIUS, useTheme } from '@aragon/ui'
+import DatePicker from './DatePicker'
+import Labels from './Labels'
+import { SINGLE_DATE } from './consts'
+import { dayjs, dateFormat } from '../../utils/date-utils'
+import handleSingleDateSelect from './utils'
+
+function SingleDatePicker({ format, startDate: startDateProp }) {
+  const theme = useTheme()
+  const labelsRef = useRef()
+  const [showPicker, setShowPicker] = useState(false)
+  const [startDate, setStartDate] = useState(startDateProp)
```

Comment:
Was thinking that probably we could drop this state, and use directly `startDate` from props since we are already having the `startDate` state in `AddEmployee`. 
What do you think?

Reply:
Yes, I also think we should be using the state from AddEmployee. Thank you for the catch!